packages/flutter_tools/templates/list_detail_app/lib/main.dart.tmpl

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
+import 'package:{{projectName}}/src/app.dart';
+import 'package:{{projectName}}/src/settings/settings_controller.dart';
+import 'package:{{projectName}}/src/settings/settings_service.dart';
```

Comment:
generally we use relative imports within a project, for many reasons, but one reason that should be sufficient here is just that it would prevent the project name from getting hardcoded into the project source, which is a huge pain.